hokusai/production.yml

```diff
@@ -84,6 +83,21 @@ spec:
                     values:
                       - foreground
 
+---
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: force-web
+  namespace: default
+spec:
+  scaleTargetRef:
+    apiVersion: extensions/v1beta1
+    kind: Deployment
+    name: force-web
+  minReplicas: 3
```

Comment:
So this means we could technically end up scaling down? Is that safe? Generally just curious if we have a small blip where traffic is smaller than usual will it scale down and potentially end up introducing latency when traffic/load increases again and we're trying to scale up.

Reply:
Good q, I would hope that the `targetCPUUtilizationPercentage` would prevent us from reaching a breaking point while scaling.  It seems like we haven't gotten down to 3 yet since implementing yesterday, it has hovered between 5-7. I'm not totally sure of the side effects maybe @izakp or @joeyAghion would have more insight here.

Reply:
Hey sorry for the delay - @eessex yeah to put it as a "breaking point" is a good way to think about it.  Think of the  `targetCPUUtilizationPercentage` as a percentage of the _requested_ CPU, so in this case, Force is requesting `700m` or `0.7` cores so that means `0.7 * 0.7 = 0.49` the autoscaler will try and keep it balanced so that the average CPU across all pods is ~1/2 a core.  Seems reasonable.